src/resources/tasks.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const { generateMethod, generateBasicMethods, RESOURCE_PATH } = require('./helpers');
+
+module.exports = (workast) => ({
+  ...generateBasicMethods(workast, RESOURCE_PATH.TASK, ['retrieve', 'update', 'del']),
```

Comment:
The basic methods are hard coded in the task resource and the `generateBasicResource` tests, if we keep it like this they will be hard coded in every new resource file. We should have them as constants in a separate file (example /resources/helpers/basic-methods.js), to access them across all files that need to generate basic methods.

Reply:
I get the idea. Basic method names are not going to change. The idea of having them as plain strings is to immediately recognize resource methods without navigating to a separate file.

Reply:
I think that since they are inside the generateBasicMethods file, you already need to open that file and check the string names for methods (example, i did not remember it was `del` instead of `delete` so i had to open and check and replicate the string). In case of a constants file you can import it and use the intellisense, that way we avoid typos across files.